app/websocket/Dockerfile

```diff
@@ -0,0 +1,10 @@
+FROM ubuntu:latest
```

Comment:
Have you tried to use the python:3.6-slim base image? ubuntu have a lot of installed unnecessary packages which will results in a bigger image, python:3.6-slim should be a good fit for that.

Reply:
Hello @youben11 it seems like a great idea!

Reply:
Done! (30af9d0 )